app/models/organisation.rb

```diff
@@ -0,0 +1,42 @@
+class Organisation
+  def self.all
+    @organisations ||= fetch_all_organisations(organisation_params)
+  end
+
+  attr_reader :content_id, :title
+
+  def initialize(attrs)
+    @content_id = attrs["content_id"]
+    @title = attrs["title"]
+  end
+
+  class << self
+    def document_type
+      "organisation"
+    end
+
+    def fetch_all_organisations(params)
+      orgs = []
+      current_page = 1
+      loop do
+        response = Services.publishing_api.get_content_items(params.merge(page: current_page))
+        orgs << response["results"].map { |attrs| Organisation.new(attrs) }
+        if response["pages"] > current_page
+          current_page += 1
+        else
```

Comment:
Will this enter an infinite loop if there's some issue with returning the orgs?

Reply:
It's a good question, I'd borrowed heavily from https://github.com/alphagov/specialist-publisher/blob/master/app/services/all_documents_finder.rb#L35 
I could put in some 'fuse' condition to stop after a finite number of iterations.